stdlib/xml/dom/NodeFilter.pyi

```diff
@@ -1,22 +1,22 @@
-from typing import Literal
+from typing import Final
 from xml.dom.minidom import Node
 
 class NodeFilter:
-    FILTER_ACCEPT: Literal[1]
-    FILTER_REJECT: Literal[2]
-    FILTER_SKIP: Literal[3]
+    FILTER_ACCEPT: Final = 1
+    FILTER_REJECT: Final = 2
+    FILTER_SKIP: Final = 3
```

Comment:
I didn't plan to add `Final` to class attributes, but since `Literal` is already used here, I changed it to `Final`